entries/s/sfu.ca.json

```
@@ -0,0 +1,19 @@
+{
+  "Simon Fraser University": {
+    "domain": "sfu.ca",
+    "tfa": [
+      "totp",
+      "custom-hardware"
+    ],
+    "custom-hardware": [
+      "HyperOTP token from the Bookstore or IT Services"
```

As the value of the custom-hardware will be searchable on 2fa.directory, it's better to just use the name of the hardware token. If you feel that it's necessary to point out where the hardware token can be acquired you can use that note for that.
```suggestion
      "HyperOTP"
```